### PR TITLE
chore(tile-converter): improve tile converter error message

### PR DESCRIPTION
**Pull request by kirillovzah**
Issue https://github.com/visgl/loaders.gl/issues/1539

### CODE REVIEW

File: modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts [hunk 1]
```diff
@@ -48,6 +49,7 @@ export default class Tiles3DConverter {
     this.sourceTileset = null;
     this.attributeStorageInfo = null;
     this.workerSource = {};
+    this.message = 'Tile converter does not work in browser, only in node js environment';
```
Comment (dryabinin-actionengine) [inline modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts:52]: I am not sure constructor of  Tiles3DConverter is a good place for such messages. 
I think separate constant will be sufficient.

File: modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ import {
 import {getWorkerURL, WorkerFarm} from '@loaders.gl/worker-utils';
 
 const I3S = 'I3S';
+const ERROR_MESSAGE = 'Tile converter does not work in browser, only in node js environment';
```
Comment (belom88) [inline modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts:28]:
```suggestion
const ERROR_MESSAGE = 'Tile converter does not work in browser, only in NodeJS environment';
```

Comment (belom88) [reply modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts:28]: Move it to `constants.ts` file in the root of the module. We have the same message in 2 files.

File: modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts [hunk 3]
```diff
@@ -66,57 +67,62 @@ export default class Tiles3DConverter {
     maxDepth: number;
     egmFilePath: string;
   }): Promise<any> {
-    const {inputUrl, outputPath, tilesetName, maxDepth, egmFilePath} = options;
-    this.conversionStartTime = process.hrtime();
-    this.options = {maxDepth};
+    if (!isBrowser) {
```
Comment (belom88) [inline modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts:70]:
```
if (isBrowser) {
  console.log(ERROR_MESSAGE);
  return ERROR_MESSAGE;
}

// Put old code here
```

File: modules/tile-converter/src/i3s-converter/i3s-converter.ts [hunk 4]
```diff
@@ -151,60 +152,66 @@ export default class I3SConverter {
...
-    this.writeQueue.startListening();
-
-    console.log('Loading egm file...'); // eslint-disable-line
-    this.geoidHeightModel = await load(egmFilePath, PGMLoader);
-    console.log('Loading egm file completed!'); // eslint-disable-line
-
-    if (slpk) {
-      this.nodePages.useWriteFunction(writeFileForSlpk);
-    }
+    if (!isBrowser) {
```
Comment (belom88) [inline modules/tile-converter/src/i3s-converter/i3s-converter.ts:155]:
```
if (isBrowser) {
  console.log(ERROR_MESSAGE);
  return ERROR_MESSAGE;
}

// Put old code here
```

File: modules/tile-converter/test/i3s-converter/i3s-converter.spec.js [hunk 5]
```diff
@@ -29,23 +29,24 @@ setLoaderOptions({
 });
 
 test('tile-converter - Converters#converts 3d-tiles tileset to i3s tileset', async (t) => {
```
Comment (belom88) [inline modules/tile-converter/test/i3s-converter/i3s-converter.spec.js:31]: Make a test for Tiles3DConverter (only for browser)

**Review by belom88 — CHANGES_REQUESTED** [review]

**Review by ibgreen — APPROVED** [review]

File: test/modules.js [hunk 6]
```diff
@@ -66,9 +66,5 @@ import '@loaders.gl/compression/test';
...
 import '@loaders.gl/zip/test';
 
-// Cli
-export const isBrowser =
-  typeof process !== 'object' || String(process) !== '[object process]' || process.browser;
-if (!isBrowser) {
-  require('@loaders.gl/tile-converter/test');
-}
+// Tile converter
+import '@loaders.gl/tile-converter/test'
```
Comment (belom88) [inline test/modules.js:70]:
```suggestion
import '@loaders.gl/tile-converter/test';
```

**Review by belom88 — APPROVED** [review]

**Review by belom88 — CHANGES_REQUESTED** [review]
Fix conflicts with master. If need help, reach out to me.

**Review by belom88 — APPROVED** [review]

**Pull request merged**